CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to Brick-a-Pic
+
+Welcome to the Brick-a-Pic project! Thank you for considering contributing. If you have any questions, please join our [Gitter]() or [open an issue]().
```

Comment:
Looks like these are missing links? Also ESlint under "Code style"

Reply:
Whoops, I guess those didn't make it in. Will do.

Reply:
Added links.